downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -201,6 +201,11 @@ Generating a new token will invalidate the existing token.
 Default = 63072000 seconds, or two years.
 | *`nginx_tls_protocols`* | Defines support for `ssl_protocols` in Nginx.
 
+Values available `TLSv1.2`, `TLSv1.3`
+
+If `nginx_tls-protocols` is set to `TLSv1.3` only TLSv1.3  is enabled.
```

Comment:
```suggestion
If `nginx_tls_protocols` is set to `TLSv1.3` only TLSv1.3  is enabled.
```

Note that for a single value like `TLSv1.3` then the `nginx_tls_protocols` variable is still a list so it should be set like `nginx_tls_protocols=['TLSv1.3']`

or in yaml format:
```yaml
nginx_tls_protocols:
  - TLSv1.3
```